README.md

```diff
@@ -3,6 +3,29 @@
 Provides a collection of API's to support deploying and serving of serverless
 applications and functions.
 
+## Requirements
+
+For the scripts in this repository to work properly, the following requirements apply.
+
+- `podman` aliased to `docker` or `docker` (17.05 or newer) (only if you need/want to
+build images from code)
+- `bash` (4.0.0 or newer)
+- `make`
+- `yq` (3.4.0)
+
```

Comment:
I found out that also GNU sed was required sometimes, not sure if this is the case in this repo.